build.sbt

```diff
@@ -102,6 +103,14 @@ lazy val couchbase =
                  whitesourceGroup := Whitesource.Group.Supported)
 
 lazy val csv = alpakkaProject("csv", "csv", Dependencies.Csv, whitesourceGroup := Whitesource.Group.Supported)
+  .settings(
+    mimaBinaryIssueFilters += ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.alpakka.csv.impl.CsvParser#FieldBuilder.this")
```

Comment:
Seems safe to suppress the removal of a constructor on a private class? Would be great if someone could confirm.

https://github.com/akka/alpakka/blob/0baab3fba160f9832b28eb430a4711b86b2373c6/csv/src/main/scala/akka/stream/alpakka/csv/impl/CsvParser.scala#L81

Reply:
These excludes go into files instead. 

Add `csv/src/main/mima-filters/1.0.x.backwards.excludes` with content

```
# Allow changes to impl
ProblemFilters.exclude[Problem]("akka.stream.alpakka.csv.impl.*")
```

instead.